utils/utils.js

```diff
@@ -0,0 +1,19 @@
+const { writeFileSync } = require('fs');
+const { join } = require('path');
+
+module.exports = {
+	async isOwner(user) {
+		const app = user.client.application;
+		await app.fetch();
+		// console.log(client);
+		if (app.owner?.ownerId) {
+			return app.owner.members.has(user.id);
+		}
+		return app.owner.id === user.id;
+	},
+
+	updateConfig(config) {
+		writeFileSync(join(__dirname, '..', 'config.json'), JSON.stringify(config, null, '\t'));
+		delete require.cache[require.resolve('../config.json')];

```

Comment:
given the "cache" in this line I assume writing to a json file results in a backup/temp file of some sort?

Reply:
When you `require()` a file it adds it to the require cache so that other parts of the program can `require` it again more quickly. Line 17 is removing the config from the require cache since its changed and the cached version is no longer accurate. It will be re-cahed as soon as it gets required again.